modules/aws-eks-teams/main.tf

```diff
@@ -98,29 +119,30 @@ resource "kubernetes_role" "team" {
   }
 }
 
-resource "kubernetes_role_binding" "team" {
+resource "kubernetes_role_binding" "this" {
   for_each = var.application_teams
+
   metadata {
     name      = "${each.key}-role-binding"
-    namespace = kubernetes_namespace.team[each.key].metadata[0].name
+    namespace = kubernetes_namespace.this[each.key].metadata[0].name
   }
+
   role_ref {
     api_group = "rbac.authorization.k8s.io"
     kind      = "Role"
     name      = "${each.key}-role"
   }
   subject {
+    api_group = "rbac.authorization.k8s.io"
     kind      = "Group"
     name      = "${each.key}-group"
-    api_group = "rbac.authorization.k8s.io"
-    namespace = kubernetes_namespace.team[each.key].metadata[0].name
```

Comment:
Namespace not required for group subject anymore? it was never needed?

Reply:
yes